### PR TITLE
[candlepin/foreman] call psql with --no-password to avoid waiting on input

### PR DESCRIPTION
**Pull request by jjansky1**
Resolved: #2429
Related: #2422
Closes: #2421

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] If this commit closes an existing issue, is the line `Closes: #ISSUENUMBER` included in an independent line?
- [X] If this commit resolves an existing pull request, is the line `Resolves: #PRNUMBER` included in an independent line?

### CODE REVIEW

File: sos/plugins/foreman.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def build_query_cmd(self, query, csv=False):
         a large amount of quoting in sos logs referencing the command being run
         """
         csvformat = "-A -F , -X" if csv else ""
-        _dbcmd = "psql -h %s -p 5432 -U foreman -d foreman %s -c %s"
+        _dbcmd = "psql --no-password -h %s -p 5432 -U foreman -d foreman %s -c %s"
```
Comment (pmoravec) [inline sos/plugins/foreman.py:264]: Line 82 chars long, per https://travis-ci.org/github/sosreport/sos/jobs/760936097 .

Comment (jjansky1) [reply sos/plugins/foreman.py]: new commit

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]

**Comment by TurboTurtle** [comment]
Merged into `legacy-3.9` via ea9be29

**Comment by jjansky1** [comment]
Hello,
it seems only commit below was merged

foreman: call psql with --no-password to avoid waiting on input (a76f088a6ef6f312abafb0ab734b5f9d096ac25e)

can i get please merged also commit below?

candlepin: call psql with --no-password to avoid waiting on input (21360a34c4003bfe40cd2ff76ecfe665c4bf0aea)

Thank you in advance (sorry for missing [] on plugin name in commit message)
JJ

**Pull request closed without merging**